.github/workflows/build_and_test.yml

```diff
@@ -1049,7 +1124,7 @@ jobs:
           sudo install minikube-linux-amd64 /usr/local/bin/minikube
           rm minikube-linux-amd64
           # Github Action limit cpu:2, memory: 6947MB, limit to 2U6G for better resource statistic
-          minikube start --cpus 2 --memory 6144
+          minikube start --cpus 2 --memory 6144 --force
```

Comment:
Why do you touch `k8s-integration-tests` in `kinesis` PR?

Reply:
Removed.